src/components/EnterpriseApp/EnterpriseAppRoutes.jsx

```diff
@@ -29,7 +29,6 @@ const EnterpriseAppRoutes = ({
   enableContentHighlightsPage,
 }) => {
   const { canManageLearnerCredit } = useContext(EnterpriseSubsidiesContext);
-  console.log('EnterpriseAppRoutes!!!');
```

Comment:
[context] These erroneous `console.log` statements slipped through in the previous PR that merged while debugging one of the review comments. Removing the `console.log` statements in this PR.